cluster/src/main/java/com/hartwig/pipeline/cram/CramAndValidateCommands.java

```diff
@@ -0,0 +1,44 @@
+package com.hartwig.pipeline.cram;
+
+import com.google.common.collect.ImmutableList;
+import com.hartwig.pipeline.calling.command.VersionedToolCommand;
+import com.hartwig.pipeline.execution.vm.Bash;
+import com.hartwig.pipeline.execution.vm.BashCommand;
+import com.hartwig.pipeline.execution.vm.JavaClassCommand;
+import com.hartwig.pipeline.resource.Resource;
+import com.hartwig.pipeline.tools.Versions;
+
+import java.util.Collections;
+import java.util.List;
+
+public class CramAndValidateCommands {
```

Comment:
I think the Substage interface would work well for this

Reply:
Still do :)

Reply:
Sorry wasn't ignoring this suggestion, just forgot. But now that I look more closely at it, I'd have to do some gymnastics to use the substages from the batch side because it doesn't really care about samples (it deals generically with input files) and I'm not sure whether it's worth the effort. For the pipeline invocation it makes more sense but not sure whether that's compelling by itself.

Reply:
I don't think it would be too hard to remove the references to samples, but also fine to leave this for now and make that a part of another separate refactoring task.